CLI/test/ressources/mass_generation_xmldiagformat/output_expected/error.xmldiag

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<build return-code="ParsingDiagnostics">
+  <filetable>
+    <file index="1" path="input\DVZZPGM1.tcbl" />
```

Comment:
Why paths are relative paths and not absolute paths?

Reply:
Good question, to build the filetable I'm using the `Inputs` dictionary from the `AbstractErrorWriter` so it looks like we store relative or absolute paths depending on the original input format passed to CLI.
@smedilol should we change that ?

Reply:
Ok, this also to make unit tests not absolute path dependent.